package.json

```diff
@@ -18,8 +18,18 @@
     "code",
     "sharing"
   ],
+  "jest": {
+    "setupFiles": [
+      "./testSetup.js"
```

Comment:
Hm.. why start with dot? Why not `tests/testSetup.js`?

Reply:
hm why not, about to move to test folder and fix this line